src/test/java/org/cy3sbml/TestUtils.java

```diff
@@ -74,15 +74,18 @@ public static void setSystemProxyForTests() {
      * Resources in the skip set are skipped.
      * If a filter string is given only the resources matching the filter are returned.
      */
-    public static Iterable<Object[]> findResources(String resourcePath, String extension, String filter, HashSet<String> skip) {
+    public static Iterable<Object[]> findResources(String where, String resourcePath, String extension, String filter, HashSet<String> skip) {
 
         File currentDir = new File(System.getProperty("user.dir"));
         // String rootPath = new File(currentDir, resourcePath).getPath();
-        String rootPath = currentDir.getAbsolutePath() + "/src/test/resources" + resourcePath;
-
-        System.out.println("curDir:" + currentDir);
-        System.out.println("rootPath:" + rootPath);
+        String rootPath;
+        if (where.equals("main")){
+            rootPath = currentDir.getAbsolutePath() + "/src/main/resources" + resourcePath;
```

Comment:
Does this work? This should be path manipulation instead of string concatenation. This will likely break on other OS such as Windows. More robust path handnling